odoo/custom/src/private/skyviv_stock_valuation_fifo_lot/models/stock_move.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2022 Ecosoft Co., Ltd (https://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _create_in_svl(self, forced_quantity=None):
+        """ 1.Overwrite for skip check stock move - Multiple lot on the stock move is not
+        allowed for incoming transfer
+            2.Create a `stock.valuation.layer` from `self`.
+        :param forced_quantity: under some circunstances, the quantity to value is different than
+        the initial demand of the move (Default value = None)
+        """
+        layers = self.env["stock.valuation.layer"]
+        for move in self:
+            svl_vals_list = []
+            for move in self:
```

Comment:
```suggestion
            for move in self:
```
why loop self in self?

Reply:
แก้ไขแล้วครับ @Saran440